learning/tour-of-beam/frontend/lib/config/theme/theme.dart

```diff
@@ -0,0 +1,160 @@
+  outlinedButtonTheme: _getOutlineButtonTheme(
+    TobDarkThemeColors.text,
+    TobDarkThemeColors.primary,
+  ),
+  elevatedButtonTheme: _getElevatedButtonTheme(TobDarkThemeColors.primary),
+  appBarTheme: _getAppBarTheme(TobDarkThemeColors.secondaryBackground),
+);
+
+TextTheme _getTextTheme(Color textColor) {
+  return GoogleFonts.sourceSansProTextTheme(
+    const TextTheme(
+      displayLarge: _emptyTextStyle,
+      displayMedium: TextStyle(
+        fontSize: 48,
+        fontWeight: FontWeight.w900,
+      ),
+      displaySmall: TextStyle(
+        fontFamily: 'Roboto_regular',
+        fontSize: 18,
+        fontWeight: FontWeight.w400,
+      ),
+      headlineLarge: _emptyTextStyle,
+      headlineMedium: _emptyTextStyle,
+      headlineSmall: TextStyle(
+        fontSize: 12,
+        fontWeight: FontWeight.w600,
+      ),
+      titleLarge: TextStyle(
+        fontSize: 24,
+        fontWeight: FontWeight.w600,
+      ),
+      titleMedium: _emptyTextStyle,
+      titleSmall: _emptyTextStyle,
+      labelLarge: TextStyle(
+        fontSize: 16,
+        fontWeight: FontWeight.w600,
+      ),
+      labelMedium: _emptyTextStyle,
+      labelSmall: _emptyTextStyle,
+      bodyLarge: TextStyle(
+        fontSize: 24,
+        fontWeight: FontWeight.w400,
+      ),
+      bodyMedium: TextStyle(
+        fontSize: 13,
+        fontWeight: FontWeight.w400,
+      ),
+      bodySmall: _emptyTextStyle,
+    ).apply(
+      bodyColor: textColor,
+      displayColor: textColor,
+    ),
+  );
+}
+
+TextButtonThemeData _getTextButtonTheme(Color textColor) {
+  return TextButtonThemeData(
+    style: TextButton.styleFrom(
+      primary: textColor,
+      shape: _getButtonBorder(TobBorderRadius.large),
+    ),
+  );
+}
+
+OutlinedButtonThemeData _getOutlineButtonTheme(
+  Color textColor,
+  Color outlineColor,
+) {
+  return OutlinedButtonThemeData(
+    style: OutlinedButton.styleFrom(
+      primary: textColor,
+      side: BorderSide(color: outlineColor, width: 3),
+      padding: _buttonPadding,
+      shape: _getButtonBorder(TobBorderRadius.small),
+    ),
+  );
+}
+
+ElevatedButtonThemeData _getElevatedButtonTheme(Color color) {
+  return ElevatedButtonThemeData(
+    style: ElevatedButton.styleFrom(
+      onPrimary: TobColors.white,
+      primary: color,
+      padding: _buttonPadding,
+      elevation: TobSizes.size0,
+    ),
+  );
+}
+
+AppBarTheme _getAppBarTheme(Color backgroundColor) {
+  return AppBarTheme(
+    color: backgroundColor,
+    elevation: TobSizes.size1,
+    centerTitle: false,
+    toolbarHeight: TobSizes.appBarHeight,
+  );
+}
+
+const EdgeInsets _buttonPadding = EdgeInsets.symmetric(
+  vertical: TobSizes.size20,
+  horizontal: TobSizes.size40,
+);
+
+RoundedRectangleBorder _getButtonBorder(double radius) {
+  return RoundedRectangleBorder(
+    borderRadius: BorderRadius.all(
+      Radius.circular(radius),
+    ),
+  );
+}
+
+const TextStyle _emptyTextStyle = TextStyle();
```

Comment:
Do you plan adding something here? It it stays empty, it would be more clear to use `const TextStyle()` and not this constant.

Reply:
I use it to see what styles are not configured

Reply:
Then please add a `///` doc comment for it.